docker-images/jenkins/Dockerfile

```diff
@@ -15,19 +15,12 @@
 FROM jenkins/jenkins:lts
 USER root
 
-#Pre-Install Jenkins Plugins
-COPY plugins.txt /usr/share/jenkins/ref/plugins.txt
-RUN /usr/local/bin/install-plugins.sh < /usr/share/jenkins/ref/plugins.txt
-
 # Install Docker
```

Comment:
@Aaquiff shall we please follow the formatting styles and other best practices which we have adopted in official WSO2 product Dockerfiles (e.g. please see [1])?

[1]: [WSO2 API Manager `v3.0.0` Dockerfile](https://github.com/wso2/docker-apim/blob/v3.0.0.1/dockerfiles/ubuntu/apim/Dockerfile#L55-L62)

Reply:
These best practices and guidelines are well documented at [1] and [2].

[1]: [Official Docker Library image review guidelines](https://github.com/docker-library/official-images#review-guidelines)
[2]: [Best practices for writing Dockerfiles](https://docs.docker.com/develop/develop-images/dockerfile_best-practices/)

Reply:
@chirangaalwis did some changes, please verify